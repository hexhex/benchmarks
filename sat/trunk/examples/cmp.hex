smaller(A,B) :- &cmp[A,B](), A = "bar", B = "foo".
smaller(A,B) :- &cmp[A,B](), A = 1, B = 2.
smaller(A,B) :- &cmp[A,B](), A = abc, B = abe.
smaller(A,B) :- &cmp[A,B](), A = "baz", B = "bar".
smaller(A,B) :- &cmp[A,B](), A = 5, B = 4.
smaller(A,B) :- &cmp[A,B](), A = may, B = june.
smaller(A,B) :- &cmp[A,B](), A = "test", B = "test".


