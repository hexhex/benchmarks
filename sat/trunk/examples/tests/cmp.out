{smaller("bar","foo"), smaller(1,2), smaller(abc,abe)}
